setup/install_upstream_rust_keylime/test.sh

```diff
@@ -7,6 +7,9 @@
 [ -n "${RUST_KEYLIME_UPSTREAM_URL}" ] || RUST_KEYLIME_UPSTREAM_URL="https://github.com/keylime/rust-keylime.git"
 [ -n "${RUST_KEYLIME_UPSTREAM_BRANCH}" ] || RUST_KEYLIME_UPSTREAM_BRANCH="master"
 
+
+COVERAGE_DIR='/var/tmp/limeLib/coverage/'
```

Comment:
Do not define the variable but rather use `$__INTERNAL_limeCoverageDir` defined by `lime`library.  Yeah, we should probably strip the `__INTERNAL_` prefix.